challenges/classes.js

```diff
@@ -1,5 +1,39 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(height, width, length) {
+    this.height = height;
+    this.width = width;
+    this.length = length;
+  }
+  // Getter
+  get volume() {
+    return this.calcArea();
+  }
+  // Method
+  calcArea() {
+    return this.height * this.width * this.length;
+  }
+  // Getter
+  get surfaceArea() {
+    return this.calcSurface();
+  }
+  // Method
+  surfaceArea() {
+    return (
+      2 *
+      (this.length * this.width +
+        this.length * this.height +
+        this.width * this.height)
+    );
```

Comment:
consider refactoring for a cleaner flow. while it does work and it does make sense, I think you can make it look better.